tests/smoketest/smoketest.sh

```diff
@@ -6,25 +6,35 @@
 #   * oc tools pointing at your STF instance
 #   * gnu sed
 #
-# Usage: ./smoketest.sh [NUMCLOUDS]
+# Usage: NUMCLOUDS=<num clouds> OCP_PROJECT=<project> ./smoketest.sh [NUMCLOUDS]
```

Comment:
I'm confused by this change. The script uses a parameter passed to it with a default of 1. Seems like you're basically just overriding that? See line 16 for where it is set. I'm not sure this is necessary? (I actually like env param better than the argument method fwiw.)

Reply:
That looks like a relic from the backport. Someone else must have made that change. Not sure why that was ever changed

Reply:
Looks like Chris made this change over a year ago when he imported changes from the old repo
```
commit f34637b5d45e59b21c21d81f092286b9955cfc75 (HEAD)
Author: Chris Sibbitt <csibbitt@redhat.com>
Date:   Fri Jan 24 17:10:27 2020 -0500

    Importing content from deprecated telemetry-framework repo
    
    * From an up to date master of both
    ```
    $ cp -a ../telemetry-framework/deploy/ .
    $ rm deploy/configs/es-workaround.bash  # We don't need this anymore
    $ cp -a ../telemetry-framework/tests/ .
    $ git add -A
    ```
    * Merged important content from README.md
    
    History before this point may be available at
    https://github.com/redhat-service-assurance/telemetry-framework
```
 I'm willing to switch it back over, but this is a change that will also need to be made in 1.3 and master. I'll include it in a new patch that I propose there and backport to here again